### PR TITLE
pr-template added 

### PR DESCRIPTION
**Pull request by MehakMantri**
Issue no- #20 

Summary
The PR template includes sections for summary, changes, testing, checklist, and additional comments, making it easier for contributors to provide structured information about their PRs. This enhancement will help maintainers and contributors communicate more effectively during the PR review process.

### CODE REVIEW

**Comment by purohitdheeraj** [comment]
Nice @MehakMantri just updated my PR for contributing guidelines according to this :D

File: PULL_REQUEST_TEMPLATE.md.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [List the main changes introduced by this PR. Use bullet points for clarity.]
+
+**Testing:**
+
+[Describe any testing you've conducted to verify that the changes are working as expected. Include steps, test data, or other relevant information.]
+
+**Checklist:**
+
+- [ ] The code follows the project's coding standards and conventions.
+- [ ] All tests have passed.
```
Comment (Neha) [inline PULL_REQUEST_TEMPLATE.md.md:18]: We don't have any tests as of now. Please change it to have you self tested the changes

File: PULL_REQUEST_TEMPLATE.md.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] Documentation has been updated, if necessary.
+- [ ] The PR has been reviewed and approved by at least one other contributor.
```
Comment (Neha) [inline PULL_REQUEST_TEMPLATE.md.md:20]: We don't reviewed and approved here as it is PR where they are requesting the same

File: index.html [hunk 3]
```diff
@@ -269,7 +269,7 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="https://static.vecteezy.com/system/resources/previews/022/100/816/non_2x/microsoft-logo-transparent-free-png.png"
```
Comment (Neha) [inline index.html:272]: Why this change is here?

**Review by Neha — COMMENTED** [review]
I have left a few small comments. Please check.

**Comment by MehakMantri** [comment]
I have updated the code with desired changes,please check.

**Comment by Neha** [comment]
This looks @MehakMantri. Thank you for contribution.
